cmake/libpython/CMakeLists.txt

```diff
@@ -24,6 +27,8 @@ if(UNIX)
             PYTHONPATH="${PYTHONPATH}"
       )
 elseif(WIN32)
+    # needed for python 3.5
+    add_definitions(-DPY3_DLLNAME="python3${CMAKE_DEBUG_POSTFIX}")
```

Comment:
This line makes PY3_DLL_NAME has debug postfix even in release configurations.

Reply:
@gongminmin Thanks for pointing this out :pray:  For future reference, the corresponding change was integrated as part of https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/pull/286